WordPress/Classes/Utility/CoreDataHelper.swift

```diff
@@ -156,7 +156,7 @@ extension NSPersistentStoreCoordinator {
     /// - Returns: a NSManagedObjectID if the uri is valid or nil if not.
     ///
     public func safeManagedObjectID(forURIRepresentation uri: URL) -> NSManagedObjectID? {
-        guard let host = uri.host, host == "x-coredata" else {
+        guard let scheme = uri.scheme, scheme == "x-coredata" else {
```

Comment:
whoa, did this ever work?

Reply:
I don't think so! I guess we don't suffer a lot of bad connections or else this should have been evident for a long time.